src/main/java/com/pokegoapi/api/map/MapObjects.java

```diff
@@ -184,15 +184,18 @@ public void update(MapObjects other) {
 
 		spawnPoints.clear();
 		addSpawnPoints(other.getSpawnPoints());
-
-		for (FortData otherGym : other.getGyms()) {
-			for (FortData gym : getGyms()) {
-				if (otherGym.getId().equals(gym.getId())) {
-					gyms.remove(gym);
-					break;
-				}
+                
+                
+                for (FortData otherGym : other.getGyms()) {
+                    Iterator<FortData> iterator = gyms.iterator();
+                    while(iterator.hasNext()){
```

Comment:
i dont think this pass the checkstyle, could you please verify with running `gradle checkstyleMain` and correct them ?